miden-lib/src/accounts/mod.rs

```diff
@@ -1,4 +1,6 @@
-use super::{auth::AuthScheme, transaction::TransactionKernel};
+use super::auth::AuthScheme;
 
+pub mod auth;
+pub mod components;
```

Comment:
Made it `pub(super)` instead since its only used in that module.

This was basically a copy of `miden-lib/src/notes/scripts.rs`. I made that module `pub(super)` as well - let me know if this is not a good idea.